### DISCUSSION TITLE
Looking for command to rollback specific table and migrate that table again

### DISCUSSION THREAD

**Discussion by JaberWiki** [nWidart/laravel-modules#1422] [Q&A]
How to rollback migration with step  or --path like default Laravel rollback commands on Laravel modules?
I've tried this command

`php artisan migrate:rollback --path=/Modules/ModuleName/Database/Migrations/table_name.php`

**or**

`php artisan module:migrate-rollback --path=/Modules/ModuleName/Database/Migrations/table_name.php Blog`

But it's not working.
Is there any command to migrate or rollback specific table?

**Comment by ai-sakib** [comment]
I was also looking for this kind of command in Laravel Module package.

**Reply by JaberWiki** [reply]
Yes it will help in production mode.

**Reply by ai-sakib** [reply]
Yeah
